client/tests/unit/serializers/application-test.js

```diff
@@ -0,0 +1,14 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Serializer | application', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
```

Comment:
Does this need to be:
```js
async function(assert)
```
Or are the two methods being called on lines 9 & 10 synchronous?